client/src/components/homepage/About.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useTranslation, Trans } from 'react-i18next';
+
+import './About.css';
+
+function About() {
+    const { t } = useTranslation();
+    return (
+        <section className="homepage__about container">
+            <div className="about__box">
+                {' '}
+                <Link className="about__hdl" to="/quizzes">
+                    {t('about.quizzes')}
+                </Link>
+            </div>
+            <div className="about__box">
+                <Link className="about__hdl" to="/calculators">
+                    {t('about.calculators')}
+                </Link>
+            </div>
+            <div className="about__box notactive">
+                <Trans>
+                    <Link className="about__hdl">{t('about.general')}</Link>
+                </Trans>
+            </div>
+            <div className="about__box notactive">
+                <Trans>
+                    <Link className="about__hdl">{t('about.general')}</Link>
+                </Trans>
+            </div>
+            <div className="about__box notactive">
+                <Trans>
+                    <Link className="about__hdl">{t('about.general')}</Link>
+                </Trans>
+            </div>
+            <div className="about__box notactive">
+                <Trans>
+                    <Link className="about__hdl">{t('about.general')}</Link>
+                </Trans>
+            </div>
+        </section>
```

Comment:
This kind of repetitive code can usually be replaced by an array with data that is mapped to jsx - for example: 

```suggestion
<section className="homepage__about container">
    {[
        { to: '/quizzes', label: t('about.quizzes'), active: true },
        { to: '/calculators', label: t('about.calculators'), active: true },
        { to: '/', label: t('about.general'), active: false },
        { to: '/', label: t('about.general'), active: false },
        { to: '/', label: t('about.general'), active: false },
        { to: '/', label: t('about.general'), active: false }
    ].map((item, idx) => (
        <div
            key={idx}
            className={`about__box${item.active ? '' : ' notactive'}`}
        >
            <Link className="about__hdl" to={item.to}>
                {item.label}
            </Link>
        </div>
    ))}
</section>
```